tests/run.sh

```diff
@@ -211,7 +221,10 @@ for f in $tests; do
     DISPLAY=$D "$AWESOME_CLIENT" 2>&1 < "$f"
 
     # Tail the log and quit, when awesome quits.
-    tail -n 100000 -s 0.1 -f --pid "$awesome_pid" "$awesome_log"
+    tail -n 100000 -s 0.1 -f --pid "$awesome_pid" "$awesome_log" \
+        | grep -vE '^.{19} W: awesome: a_dbus_connect:[0-9]+: Could not
+    connect to D-Bus system bus:' \
+        | grep -v '^Test finished successfully\.' || true
```

Comment:
The dot is meant to be matched literal, and needs to be escaped then (also without -E), no?